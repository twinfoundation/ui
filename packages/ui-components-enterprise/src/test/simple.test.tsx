// Copyright 2024 IOTA Stiftung.
// SPDX-License-Identifier: Apache-2.0.

import { render } from '@testing-library/react';
import { describe, it, expect } from 'vitest';

// Simple test component to verify testing setup works
const SimpleComponent = ({ text }: { text: string }) => {
  return <div data-testid="simple-component">{text}</div>;
};

describe('Simple Component', () => {
  it('renders correctly', () => {
    const { getByTestId } = render(<SimpleComponent text="Hello World" />);
    expect(getByTestId('simple-component')).toHaveTextContent('Hello World');
  });

  it('matches snapshot', () => {
    const { container } = render(<SimpleComponent text="Test" />);
    expect(container).toMatchSnapshot();
  });
}); 