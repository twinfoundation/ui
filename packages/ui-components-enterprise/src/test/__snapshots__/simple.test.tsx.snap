// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Simple Component > matches snapshot 1`] = `
<div>
  <div
    data-testid="simple-component"
  >
    Test
  </div>
</div>
`;
